src/UserStories/UserStories.css

```diff
@@ -8,31 +8,36 @@ html {
   overflow: hidden;
 }
 .stories-scrollable {
+  transition: transform 0.2s ease-out;
```

Comment:
This is the cause of the weird dance when you flip to the next story. I'd change it to `0.4s` to make it slower than the scroll effect of `stories-container` (which I'd change to `0.2s`).

Reply:
Done